crates/illumos-sys-hdrs/src/kernel.rs

```diff
@@ -211,12 +211,14 @@ unsafe impl Sync for cb_ops {}
 pub struct dev_ops {
     pub devo_rev: c_int,
     pub devo_refcnt: c_int,
-    pub devo_getinfo: unsafe extern "C" fn(
-        dip: *mut dev_info,
-        infocmd: ddi_info_cmd_t,
-        arg: *mut c_void,
-        result: *mut *mut c_void,
-    ) -> c_int,
+    pub devo_getinfo: Option<
```

Comment:
I know this is technically a more accurate representation, but we don't actually set it to `None`/`NULL` anymore and realistically this crate is only used by xde. So I'm a bit inclined to revert it?

Reply:
I wasn't sure if we wanted to keep it around in case `mac` registration were to make its way into illumos-rs or another crate of that nature -- although that work would probably be undertaken with a close eye wrt the manpages.